Python/Algorithms/data_structures/list_structs/fifo_lifo.py

```diff
@@ -0,0 +1,87 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Abstract_FIFO_LIFO:
+    
+    @abstractmethod
+    def insert(self,value):
+        return
+    
+    @abstractmethod
+    def remove(self,value):
+        return
```

Comment:
This is common for Any linked list. By the way, do not change the current Design. Let it be as it is. Keep the directory structure and content as it is. Just **Debug and Fix**

Reply:
This is for you to learn as to how you need to perform design of your data structures and algorithms

Reply:
I changed the names from `FIFO_LIFO` to `List` as this was the list implementation. I have tried to follow the same approach as the example given for linked lists.